parse_whos_home_output.py

```diff
@@ -32,6 +32,13 @@ def create_db(conn):
     mac INTEGER NOT NULL,
     name TEXT NOT NULL
 )''')
+    # Create notes table
+    c.execute('''
```

Comment:
We could change these to `CREATE TABLE IF NOT EXISTS` functions, then make them run every time. This would make migration invisible (at least for this)

Reply:
Added a migrate parameter